docs/en/getting-started/install.md

```diff
@@ -99,7 +99,7 @@ It is recommended to use official pre-compiled `deb` packages for Debian or Ubun
 sudo apt-get install -y apt-transport-https ca-certificates dirmngr
 sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 8919F6BD2B48D754
 
-echo "deb https://packages.clickhouse.com/deb stable main" | sudo tee \
+echo "deb [arch=amd64] https://packages.clickhouse.com/deb stable main" | sudo tee \
```

Comment:
But we support no only amd64.
It should work for AArch64 as well.

Reply:
That's right. Maybe we can modify the description with `echo "deb [arch=$(dpkg --print-architecture)] https://packages.clickhouse.com/deb stable main"` command.

And it can get the CPU architecture in the current machine.